bridges/otelzerolog/hook.go

```diff
@@ -100,5 +102,34 @@ func NewHook(name string, options ...Option) *Hook {
 
 // Run handles the passed record, and sends it to OpenTelemetry.
 func (h Hook) Run(e *zerolog.Event, level zerolog.Level, msg string) {
-	// TODO
+	r := log.Record{}
+	r.SetSeverity(convertLevel(level))
+	r.SetBody(log.StringValue(msg))
+	r.SetTimestamp(time.Now())
```

Comment:
```suggestion
```

This is not a value coming from zerolog. You can create an issue similar to https://github.com/rs/zerolog/issues/493.

The API implementations can set timestamps if they are unset.
For instance the OTel Go SDK set `ObservedTimestamp` to `time.Now()`.
See: https://github.com/open-telemetry/opentelemetry-go/blob/5fef3eb220acc4691497da329850db5daf0c2a3f/sdk/log/logger.go#L75-L78